### DISCUSSION TITLE
group-odd and group-even

### DISCUSSION THREAD

**Discussion by dolanmiu** [tailwindlabs/tailwindcss#3448] [Help] [answered]
I am trying to have an effect where for every odd child, I would like to apply a style to it's own children. `group-hover` works wonderfully, and I am wondering if a similar feature, `group-odd` or `group-even` could be added?

https://tailwindcss.com/docs/hover-focus-and-other-states#odd-child

Use case:

```html
<tr class="group odd:bg-gray-100">
    <td>
        <span class="group-odd:text-red-400">...</span>
    </td>
</tr>
<tr class="group odd:bg-gray-100">
    <td>
        <span class="group-odd:text-red-400">...</span>
    </td>
</tr>
<tr class="group odd:bg-gray-100">
    <td>
        <span class="group-odd:text-red-400">...</span>
    </td>
</tr>
<tr class="group odd:bg-gray-100">
    <td>
        <span class="group-odd:text-red-400">...</span>
    </td>
</tr>
<tr class="group odd:bg-gray-100">
    <td>
        <span class="group-odd:text-red-400">...</span>
    </td>
</tr>
```

**Comment by stefandb** [comment, marked as answer]
HI dolanmiu

I had the same question and made a plugin for this. Here is the npmjs.com link: https://www.npmjs.com/package/tailwind-group-even-odd

**Comment by miguelvictor** [comment]
no planned support for this ? 🙂

**Comment by root5427** [comment]
Any updates on this?

**Reply by adamwathan** [reply]
This just works in v3+ since `group-*` works with everything now:

https://play.tailwindcss.com/rbs4vD0kvH
